docs/ref/django-admin.txt

```diff
@@ -805,6 +805,20 @@ The behavior of this command changes depending on the arguments provided:
   migrated past the named migration. Use the name ``zero`` to unapply all
   migrations for an app.
 
+.. admonition:: ``django-admin migrate <app_label> zero``
+
+    The special zero ``<migration_name>`` to unapply(reverse) all the applied
+    migrations for the given ``<app_label>``.
+
+.. warning::
+
+    When unapplying any migration, all dependent migrations will also be
+    unapplied (reverted), regardless of ``<app_label>``. This is also truth
+    for ``migrate <app_label> zero``, so use it with caution.
+
+    You can always use ``--plan`` to check the migration plan and and prevent
+    any unwanted propagation to other apps.
```

Comment:
This warning is not a part of the ticket solution (but I agree that it's a good improvement), please move it to a separate commit. I also think that we don't need a special warning for a `zero` migration, so maybe:
```
.. warning:

    Be aware that when unapplying migrations all dependent migrations will also
    be reverted, regardless of ``<app_label>``. You can use ``--plan`` to check
    which migrations will be reverted.
```

Reply:
Hi just created a new PR just for this warning pull/11842

Reply:
Thanks!